support/e2e/package.json

```diff
@@ -8,6 +8,7 @@
     "chromium": "REMIRROR_E2E_BROWSER=chromium jest --runInBand --verbose",
     "chromium:watch": "REMIRROR_E2E_BROWSER=chromium jest --verbose --watch",
     "firefox": "REMIRROR_E2E_BROWSER=firefox jest --runInBand --verbose",
+    "root-cause": "root-cause",
```

Comment:
Is there a way exec bins without it?
(yarn / npx equivalent)

Reply:
`pnpx root-cause` might work